admin/class-admin.php

```diff
@@ -14,14 +14,11 @@ class WPSEO_Admin {
 	private $options;
 
 	/**
-	 * @var Yoast_Dashboard_Widget
-	 */
-	public $dashboard_widget;
-
-	/**
-	 * @var WPSEO_GSC
+	 * Array of classes that add admin functionality
+	 *
+	 * @var array
 	 */
-	private $page_gsc;
+	public $admin_features;
```

Comment:
Why is this public? It's now possible to access this from outside and break functionality.